frontend/packages/console-app/src/components/guided-tours/GuidedTourCatalog.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+import { GuidedTourCatalogItem } from './utils/guided-tour-typings';
+import { Gallery } from '@patternfly/react-core';
+import GuidedTourItem from './GuidedTourItem';
+import './GuidedTourCatalog.scss';
+
+type GuidedTourCatalogProps = {
+  tours: GuidedTourCatalogItem[];
+};
+
+const GuidedTourCatalog: React.FC<GuidedTourCatalogProps> = ({ tours }) => (
+  <div className="odc-guided-tour__catalog">
```

Comment:
Would like to be able to use PF `PageSection` for the background coloring although I don't know if that's possible with the current console layout. Give it a shot. Otherwise this is ok.